extensions/pkg/controller/bastion/actuator.go

```diff
@@ -13,12 +13,36 @@ import (
 	extensionsv1alpha1 "github.com/gardener/gardener/pkg/apis/extensions/v1alpha1"
 )
 
-// Actuator acts upon Bastion resources.
+// Actuator acts upon [extensionsv1alpha1.Bastion] resources.
 type Actuator interface {
-	// Reconcile reconciles the Bastion.
+	// Reconcile reconciles the [extensionsv1alpha1.Bastion] resource.
+	//
+	// Implementations of the [Actuator] interface should take care for
+	// creating or updating the bastion host managed by the extension, so
+	// that it is brought to its desired state.
 	Reconcile(context.Context, logr.Logger, *extensionsv1alpha1.Bastion, *extensionscontroller.Cluster) error
-	// Delete deletes the Bastion.
+
+	// Delete is invoked when the [extensionsv1alpha1.Bastion] resource is
+	// deleted.
+	//
+	// Implementations of the [Actuator] interface should take care of
```

Comment:
```suggestion
	// Implementations should take care of
```

To make it consistent to the `BackupBucket` (et al) doc strings, where `of the [Actuator]` is not mentioned.